package/MDAnalysis/analysis/contacts.py

```diff
@@ -584,19 +584,33 @@ def output_exists(self, force=False):
         """
         return os.path.isfile(self.output) and not (self.force or force)
 
-    def run(self, store=True, force=False, start_frame=1, end_frame=None, step_value=1):
+    def run(self, store=True, force=False, start=0, stop=None, step=1, **kwargs):
         """Analyze trajectory and produce timeseries.
 
         Stores results in :attr:`ContactAnalysis1.timeseries` (if store=True)
         and writes them to a data file. The average q is written to a second
         data file.
-        *start_frame*
-            The value of the first frame number in the trajectory to be used (default: frame 1)
-        *end_frame*
-            The value of the last frame number in the trajectory to be used (default: None -- use all frames)
-        *step_value*
+        *start*
+            The value of the first frame index in the trajectory to be used (default: index 0)
+        *stop*
+            The value of the last frame index in the trajectory to be used (default: None -- use all frames)
+        *step*
             The number of frames to skip during trajectory iteration (default: use every frame)
         """
+        import warnings
+
+        if 'start_frame' in kwargs:
+            warnings.warn("start_frame argument has been deprecated, use start instead")
+            start = kwargs['start_frame']
+
+        if 'end_frame' in kwargs:
+            warnings.warn("end_frame argument has been deprecated, use stop instead")
+            stop = kwargs['end_frame']
+
+        if 'step_value' in kwargs:
+            warnings.warn("step_value argument has been deprecated, use step instead")
+            step = kwargs['step_value']
+
         if self.output_exists(force=force):
             import warnings
```

Comment:
The import is already at the top of the module (I am well aware that imports generally belong at the top of modules). Look at line 615--you've already had to import `warnings` inside the class method despite its previous top-level import. While this is not ideal, it is a separate issue and if you remove the class method level import while feeding in a deprecated argument to `run()` you will get an `UnboundLocalError`. 

You did the same thing for the other class in this module as well, so whatever redesigns are needed for the import mechanics to work should be done elsewhere as other classes are affected too.